### ISSUE TITLE
Translation: Khmer (Cambodia)

### ISSUE THREAD

**Issue by vsay01** [jwasham/coding-interview-university#520]
This is the discussion thread for Khmer translation.
I will go from top to bottom, and section by section.

If anyone is interested, please response to this thread to finalize workflow.

**Comment by vsay01** [comment]
So far I translated headers, content, what is it, why use it and how to use it sections.

https://github.com/vsay01/coding-interview-university/blob/develop/translations/README-kh.md

**Comment by vsay01** [comment]
Update: I translated until Arrays section under Data Structure.

https://github.com/vsay01/coding-interview-university/blob/develop/translations/README-kh.md

**Comment by vsay01** [comment]
@jwasham I finished the translation and made a PR https://github.com/jwasham/coding-interview-university/pull/676

**Comment by jwasham** [comment]
I can't read it so I don't know if it's correct :)
Can see if someone reviews.

**Comment by vsay01** [comment]
@jwasham yep, we can leave it open for the time being and see if there's anyone can review this. I didn't translate the technical words because of two main reasons:

1. Those technical words in Khmer are not well-known, so some people might know and some not. Also, people might translate these words differently since it's more foreign words than native words.
2. I think it's more beneficial for Cambodian people to know English technical words so that when they go to different resources, they can familiar with those
